setup.py

```diff
@@ -21,8 +21,8 @@
     keywords=["pafy", "API", "YouTube", "youtube", "download", "video"],
     author="np1",
     author_email="np1nagev@gmail.com",
-    url="http://np1.github.io/pafy/",
-    download_url="https://github.com/np1/pafy/tarball/master",
+    url="https://github.com/mps-youtube/pafy/",
+    download_url="https://github.com/mps-youtube/pafy/tarball/master",
```

Comment:
Previously, this download URL pointed to the tarball of the master branch which held all the changes till last release. Now the master branch has been deleted since it was considered to an unnecessary burden to pull changes whenever there was a new release and since we already have GitHub tags to keep track of releases (also see #244).

But since PyPI already hosts release copies for download anyway, we could change this external download URL to point towards a list of GitHub (project homepage) tags https://github.com/mps-youtube/pafy/tags. That is what big projects like [tensorflow do](https://pypi.org/project/tensorflow/) and I think it makes some sense.

Reply:
I'm fine with that. But wouldn't you prefer if the `download_url` pointed to the release page instead? https://github.com/mps-youtube/pafy/releases

Reply:
No strong reasons but I simply seemed to like the compact listing of versions on the tags page.